packages/labs/ssr-dom-shim/src/index.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
```

Comment:
Should the date reflect when the file was made or when the code was written? If latter can ignore.
```suggestion
 * Copyright 2022 Google LLC
```

Reply:
From go/copyright:

```
the year the file was first published in full numeric form (e.g., 2010);
don't change this if you edit, move, or copy the file.
```

I consider this file to have been copied, and then edited, so I think the year stays the same.